src/Microsoft.CSharp/tests/RuntimeBinderExceptionTests.cs

```diff
@@ -112,11 +109,5 @@ public void NonTypeToCtor()
             Func<CallSite, object, object> targ = site.Target;
             AssertExtensions.Throws<ArgumentException>("Type Argument", () => targ.Invoke(site, 23));
         }
-
-        [Fact]
-        public void AssertExceptionDeserializationFails()
-        {
-            BinaryFormatterHelpers.AssertExceptionDeserializationFails<RuntimeBinderException>();
```

Comment:
Should these be replaced with valid tests now?

Reply:
It will be replaced with generic tests in System.Runtime.Serialization.Formatters.Tests that check for the correct ctor behavior